.github/workflows/ci-java.yml

```diff
@@ -0,0 +1,28 @@
+# https://docs.github.com/en/actions/guides/building-and-testing-java-with-maven
+name: Java CI
+
+on: [ push ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
Would suggest to you to attach a name to all steps. Looks nicer on GH in the end 😉